example/particles/particles.cpp

```diff
@@ -25,6 +25,16 @@
 // *************************************************//
 namespace particles_example {
 
+std::unique_ptr<ParticleBound, DeviceDeleter<parthenon::DevMemSpace>>
+SetSwarmIx1UserBC() {
+  return DeviceAllocate<ParticleBoundIX1User>();
+}
```

Comment:
Nitpicking: why the mismatch between SetSwarmI**x**1User and ParticleBoundI**X**User?

Reply:
Fixed (to `X` from `x`) :+1: